test/Grpc.NetCore.HttpClient.Tests/Grpc.NetCore.HttpClient.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="Proto\*.proto" GrpcServices="Client" Generator="MSBuild:Compile" />
+    <None Remove="@(Protobuf)" />
```

Comment:
No need for the None and Content item groups. This is only for design time build and we don't really need it for the tests.

Reply:
Meh. When we can remove them we'll remove them everywhere.